README.md

```diff
@@ -62,6 +62,24 @@ from internal types to those the API expects to return.
 This approach to pluggable interfaces is similar to other core Kubernetes APIs
 like +NetworkPolicy+, +Ingress+ and +CustomMetrics+.
 
+## Ecosystem
+
```

Comment:
Can you please define the difference between an implementation and an integration for the purposes of this addition? I think that would be useful to the reader.

Reply:
Would we want to briefly state it as: "An implementation of SMI implements the spec in a direct way, while an integration of SMI allows interoperability (defined by the spec) for tools which are not direct implementations of SMI." 

@grampelberg for insight into what he had in mind when suggesting this split.

Reply:
Something goes on top and interacts with SMI (eg. flagger). Then there's something that goes on the bottom and actually enacts what SMI asked for (eg. linkerd).

Reply:
@grampelberg I think "actually enacts" sounds like "implements", so I can see this could still be confusing. How would "enact what SMI asked for" not be "implements SMI"? When I read this comment, I think "so is linkerd an implementation? then why is it listed in integrations?" (on https://deploy-preview-7--smi-spec.netlify.com/).

Reply:
@bridgetkromhout I was just trying to use different words. You're right, we should swap the two sections.

Reply:
So, I'll add the intro by @bridgetkromhout :
> An implementation of SMI implements the spec in a direct way, while an integration of SMI allows interoperability (defined by the spec) for tools which are not direct implementations of SMI.

But could you elaborate on the swapping please, not sure if I got it completely? Are you suggesting to move the tech in implementations with integrations?

Reply:
I think we should just call them all "Tools" or "Ecosystem" - trying to split these hairs is going to lead to problems. We don't realize significant benefits from having two sections.

Reply:
That's fine for me, I'll revert the seperation

Reply:
Done